lq1/dev.cfg

```diff
@@ -39,3 +36,6 @@ echo " "
 echo l = quad
 echo " "
 echo k = kill
+
+echo [NOTICE] If u used the "c" for swim down "freeze" has now overwriten it.
+echo if u were using default LibreQuake keybinds "ctrl" key is also bound to swim down
```

Comment:
I see no edit to configuration files that changes `c` to *not* refer to `+movedown`.

Reply:
our config doesnt bind c to move down **but** the default quake config does and other players might rebind it to c as a preference

Reply:
Yes, it does. Added in this PR
![IMG_8377](https://github.com/MissLavender-LQ/LibreQuake/assets/26030250/36d5fb13-36ff-4506-82bc-31704e7d7d3a)

Reply:
oh i forgot
wait then why did u say otherwise?

Reply:
ohhh! i misread

Reply:
dev.cfg 

binds c to freeze

Reply:
One of them should probably be removed to avoid confusing users as to why their attempt to rebind `c` in default/config doesn't work

Reply:
thats why there is a warning